rts/System/Sound/OpenAL/Sound.cpp

```diff
@@ -454,9 +454,17 @@ void CSound::OpenLoopbackDevice(const std::string& deviceName)
 {
 	assert(curDevice == nullptr);
 
-	hasAlcSoftLoopBack = (alcIsExtensionPresent(nullptr, "ALC_SOFT_loopback") == AL_FALSE);
-	if (!hasAlcSoftLoopBack)
+#ifdef ALC_SOFT_loopback
```

Comment:
This should mostly be the case right? Perhaps it would be more interesting to log when it's not defined instead?